### PR TITLE
[4] Empty State for Smart Search Index page

### PR DESCRIPTION
**Pull request by PhilETaylor**
### Actual result BEFORE applying this Pull Request

Screenshot 2021-05-02 at 18 19 00

### Expected result AFTER applying this Pull Request

Screenshot 2021-05-02 at 18 18 40

// cc https://github.com/joomla/joomla-cms/issues/33266

### CODE REVIEW

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on 1d1c5e345fb1f49529e748fb16afd48cf4dc3aa6<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33489">issues.joomla.org/tracker/joomla-cms/33489</a>.</sub>

**Comment by richard67** [comment]
I've updated the branch to latest 4.0-dev to get rid of the unrelated javascript-cs error in drone and restored the previous test result in the issue tracker.

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on 94af1e6f276154e8408701a9d28416c5cda8eff8<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33489">issues.joomla.org/tracker/joomla-cms/33489</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33489">issues.joomla.org/tracker/joomla-cms/33489</a>.</sub>

File: administrator/language/en-GB/com_finder.ini [hunk 1]
```diff
@@ -108,7 +109,7 @@ COM_FINDER_HEADING_PHRASE="Search Phrase"
 COM_FINDER_HEADING_RESULTS="Results"
 COM_FINDER_HEADING_SEARCH_TERM_ASC="Search Phrase ascending"
 COM_FINDER_HEADING_SEARCH_TERM_DESC="Search Phrase descending"
-COM_FINDER_INDEX="Index"
+COM_FINDER_INDEX="Start the indexer"
```
Comment (Quy) [inline administrator/language/en-GB/com_finder.ini]: To be consistent with `Clear Index`, maybe rename this to `Run Indexer` or `Start Indexing` or `Start Indexer`.

Comment (PhilETaylor) [reply administrator/language/en-GB/com_finder.ini]: Yup

Comment (Quy) [inline administrator/language/en-GB/com_finder.ini]:
```suggestion
COM_FINDER_INDEX="Index"
```
Revert for now to merge PR. Will do a separate PR for the button renaming.

**Comment by Quy** [comment]
Thank you!

**Comment by PhilETaylor** [comment]
Thanks.
